bundle.proto

```diff
@@ -52,9 +73,14 @@ message WinningBatchBidRejected {
 message StateAuctionBidRejected {
   // Auction's unique identifier.
   string auction_id = 1;
+
   // Bundle's simulated bid.
   uint64 simulated_bid_lamports = 2;
+
   optional string msg = 3;
+
+  // This auction's winning bundle bid.
```

Comment:
```suggestion
  // This auction's winning bid in lamports.
```

Reply:
could potentially accept bids in other tokens e.g. jitoSOL, usdc etc.